airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -136,6 +137,8 @@
             "required": ["method"],
             "properties": {
               "method": {
+                "title": "",
+                "description": "",
```

Comment:
Are these fields required? I think that they won't show up in the UI any way.

Reply:
the pre-commit hook wanted me to add these. Maybe I just have it misconfigured though...

Reply:
Got it. Never mind then.